.pre-commit-config.yaml

```diff
@@ -7,3 +7,7 @@ repos:
     rev: 3.8.3
     hooks:
     - id: flake8
+      additional_dependencies: [
+          'flake8-docstrings==1.6.0',
+          'flake8-executable==2.1.1',
+      ]
```

Comment:
This should be enough, as plugins run by default after installation (according to the docstrings).

Pinning and updating the right version will be left for another PR, probably following changes in #288